### ISSUE TITLE
Glitching on plonged use

### ISSUE THREAD

**Issue by nicorikken** [G-dH/gnome-colorblind-filters#12]
I tried the plugin and having it active for multiple hours on end the whole desktop was glitching.

Situation:
- Ubuntu 22.04.2 LTS
- Gnome Shell 42.5
- Extension version 15
- Wayland
- Multiple monitors, of which one (laptop) is hi-dpi

There didn't seem to be issues with a single monitor, but with multiple monitors it seems that apps render pixelated or glitchy in a way as though a buffer with rendering frames is looping or out of order. Dragging windows around would help a bit.

Turning the extension off and on seems to have solved the issue.
I don't yet know what is going on here. If I can narrow it down further I'll report back.

Thanks for developing this extension!

**Comment by G-dH** [comment]
Thanks for the report. The glitching is something that is probably out of my reach (or at least skill), I just add the effect to the stage and the rest is on the compositor. Some combinations of gk, drivers and compositor are more prone to this glitching and multi-monitor/hi-DPI is definitely not helping..

**Comment by nicorikken** [comment]
@G-dH yeah, that was already what I thought as I briefly looked through the extension code earlier. Any suggestion where to report this upstream?

**Comment by G-dH** [comment]
Try https://gitlab.gnome.org/GNOME/gnome-shell/-/issues

**Comment by CoderRuq** [comment]
I just discovered the color correction modes in CHC-Extended which are great. Perhaps you could add in per-window color correction modes to colorblind filters as well? It could get around the whole-screen color correction glitches for those with those issues (Including me)?

**Comment by CoderRuq** [comment]
no worries! I wasn't sure if it was only the full-screen color correction that was causing the glitches.
